packages/scripts/README.md

```diff
@@ -435,13 +436,38 @@ This is how you execute those scripts using the presented setup:
 * `npm run test:unit` - runs all unit tests.
 * `npm run test:unit:help` - prints all available options to configure unit tests runner.
 * `npm run test:unit:watch` - runs all unit tests in the watch mode.
+* `npm run test:unit:debug` - runs all unit tests with the node debugger enabled.
 
 Jest will look for test files with any of the following popular naming conventions:
 
 - Files with `.js` (or `.ts`) suffix located at any level of depth in `__tests__` folders.
 - Files with `.js` (or `.ts`) suffix directly located in `test` folders.
 - Files with `.test.js` (or `.test.ts`) suffix.
 
+#### Debugging Jest unit tests
+
+Tests can be debugged in Google Chrome, Visual Studio Code, or any other [Inspector Client](https://nodejs.org/en/docs/guides/debugging-getting-started/#inspector-clients).
+
+##### Debugging in Google Chrome
+
+Place `debugger;` statements in any test and run `npm run test:unit:debug`.
+
+Then open `about:inspect` in Google Chrome and select `inspect` on your process.
+
+A breakpoint will be set at the first line of the script (this is done to give you time to open the developer tools and to prevent Jest from executing before you have time to do so). Click the resume button in the upper right panel of the dev tools to continue execution. When Jest executes the test that contains the debugger statement, execution will pause and you can examine the current scope and call stack.
+
+##### Debugging in Visual Studio Code
```

Comment:
The same question applies, what about other IDEs?

Reply:
I have a similar feeling here. The section could target IDEs in general and could include the details you shared in the comment and we would just have this example for VSC included as is. The truth is that VSC and Chrome are the most popular anyway 😃